Rlog/ViewModel/WorkSpace/WorkSpaceCreateScheduleListViewModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  WorkSpaceCreateScheduleListViewModel.swift
+//  Rlog
+//
+//  Created by 송시원 on 2022/10/20.
+//
+
+import SwiftUI
+
+final class WorkSpaceCreateScheduleListViewModel: ObservableObject {
+    @Published var isShowingModal = false
+    @Published var isShowingConfirmButton = false
+    
+    @Published var scheduleList: [Schedule] = [] {
+        didSet {
+            if !scheduleList.isEmpty {
+                isShowingConfirmButton = true
+            }
+            print("변경됨")
+            isShowingConfirmButton = true
+            print(isShowingConfirmButton)
+
+        }
+    }
+
+    
+    func didTapAddScheduleButton() {
+        showModal()
+    }
```

Comment:
위에 두칸을 띄운 이유가 있으실까요? 없으시면 한 줄 띄우는 것을 추천합니다!

```suggestion
    
    func didTapAddScheduleButton() {
        showModal()
    }
```

Reply:
없습니다! 수정하겠습니다 :)